src/app/contact/contact.component.spec.ts

```diff
@@ -33,8 +34,18 @@ describe('ContactComponent', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
-
-  it('should be created', () => {
-    expect(component).toBeTruthy();
+  describe('ngOnInit()', () => {
+    it('should set model to be a new ContactForm object', () => {
+      component.ngOnInit();
+      expect(JSON.stringify(component.model)).toBe(JSON.stringify(new ContactForm('', '', '')));
+    });
+  });
+  describe('sendEmail()', () => {
+    beforeEach(() => {
+      component.sendEmail();
+    });
+    it('should push to alerts', () => {
+      expect(component.alerts.length).toBe(1);
+    });
```

Comment:
Can we add a test to assert what is pushed to the alerts array?

>    type: 'info',
      msg: `Submitting...`

Reply:
Also, in general it would be good to test the promise resolve/error handling, but my understanding is the email service doesnt work right now anyways?? If thats the case then Im fine with leaving this until it does.

Reply:
Confirmed email stuff doesn't work currently so I'll leave it for the time being